### PR TITLE
Fix typing in getex command

### PR DESCRIPTION
**Pull request by Andrew-Chen-Wang**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `$ tox` pass with this change (including linting)?
- [ ] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Is there an example added to the examples folder (if applicable)?
- [ ] Was the change added to CHANGES file?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

_Please provide a description of the change here._

### CODE REVIEW

**Review by chayim — APPROVED** [review]

**Comment by WisdomPill** [comment]
@Andrew-Chen-Wang and @chayim could this be merged ASAP? It makes not python 3.10 unhappy :(

**Comment by chayim** [comment]
I mean sure, since I literally caught this ;). But you'll need to grab master.

**Comment by WisdomPill** [comment]
is there an issue for adding mypy?

**Comment by WisdomPill** [comment]
thanks for the quick merge... for the master, there is no problem, `django-redis` uses `latest` and `master`, it will be the first time it will pass checks on `master` but not `latest` 😂

**Comment by Andrew-Chen-Wang** [comment]
> is there an issue for adding mypy?

We're not really concerned about typing at the moment since the async portion is being reimplemented for performance sake and the sync portion of redis-py doesn't have typing yet. Once some of these things get flattened out, then I think mypy can be enabled.

TL;DR just lots of things need to get done first.
